src/Microsoft.NetFramework.Analyzers/Core/DoNotUseInsecureDtdProcessing.cs

```diff
@@ -436,31 +430,13 @@ private void AnalyzeObjectCreationForXmlDocument(OperationAnalysisContext contex
                                 }
                                 else // Non secure resolvers
                                 {
-
-                                    if (operation.Operand is IObjectCreationExpression xmlResolverObjCreated)
```

Comment:
Tagging @qinxgit , who initially wrote this analyzer and his feedback would be valuable.